MIDAS/src/hardware/Camera.cpp

```diff
@@ -0,0 +1,60 @@
+#include <ACAN2517.h>
+#include <SPI.h>
+
+static const byte MCP2517_SCK  = 0;
+static const byte MCP2517_MOSI = 0;
+static const byte MCP2517_MISO = 0;
+
+static const byte MCP2517_CS  = 0;
+static const byte MCP2517_INT = 0;
```

Comment:
we will not be using the interrupt pin. we can make the driver not look at the interrupt pin by making the pin 255. we also want to make this a macro as well